internal/codeintel/db/commits.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/keegancsmith/sqlf"
 )
 
+// HasCommit determines if the given commit is known for the given repository.
+func (db *dbImpl) HasCommit(ctx context.Context, repositoryID int, commit string) (bool, error) {
+	count, err := scanInt(db.queryRow(ctx, sqlf.Sprintf(`SELECT COUNT(*) FROM lsif_commits WHERE repository_id = %s and commit = %s`, repositoryID, commit)))
```

Comment:
suggestion:
probably inconsequential, but easy optimization i think
```suggestion
	count, err := scanInt(db.queryRow(ctx, sqlf.Sprintf(`SELECT COUNT(*) FROM lsif_commits WHERE repository_id = %s and commit = %s LIMIT 1`, repositoryID, commit)))
```

Reply:
Does this optimize anything or does it just increase readability? There should be a unique index on repositoryId/commit

Reply:
nvm I see now